### PR TITLE
fix: [PATRON2020] even more material ui 🎉 🥳

### PR DESCRIPTION
**Pull request by karlosos**
## [PATRON2020-192](https://tracker.intive.com/jira/browse/PATRON2020-192) Redesign

Acceptance criteria:
- design follows material design principles
- defined color theme (primary, secondary colors)
- changed component
  * header - change colors, add shadow, fix height
  * background - fixed to almost white
  * set primary and secondary colors (`theme.palette`)
  * correctly style modal on map (change font and background)
  * authors list - change to material-ui cards
  * use consistent typography

Additionally I've added scripts for hot reloading (frontend build after every change). Added adequate section in README. 

## Main view

image

image

## Authors list

image

image

## Modal 

image

image

### CODE REVIEW

File: frontend/src/common/constants/sensorsInfo.js [hunk 1]
```diff
@@ -1,17 +1,40 @@
...
-  windowSensorBorder: { color: '#551acc', description: 'Okno' },
-  windowBlindBorder: { color: '#b02061', description: 'Zasłony' },
-  RFIDSensorBorder: { color: '#cc5a52', description: 'RFID' },
-  smokeSensorBorder: { color: '#505050', description: 'Czujnik dymu' },
-  RGBLightBorder: { color: '#067007', description: 'Światło' }
+  temperatureSensor: {
+    color: 'HSL(30, 100%, 60%)',
+    colorLight: 'HSL(30, 100%, 90%)',
+    colorDark: 'HSL(30, 100%, 30%)',
+    description: 'Temperatura'
```
Comment (jwieckowski) [inline frontend/src/common/constants/sensorsInfo.js:6]: do we need to keep this property? I think it's not used anywhere

File: README.md [hunk 2]
```diff
@@ -61,4 +61,15 @@ var doSetCookie = function setCookie (c_name, value, exdays) {
...
+### Starting application in development environment
+
+In order to have hot reload run scripts:
+
+```
+npm run start:watch
+npm run build:frontend:dev
+```
+
+Backend wll serve new builds as frontend will be rebuilded on changes.
```
Comment (makeros) [inline README.md:75]: Typo in first *will ;)

File: src/public/authors.json [hunk 3]
```diff
@@ -9,7 +9,7 @@
     "id": 2,
     "name": "Hania",
     "github": "https://gitlab.com/parsleyyy",
-    "avatar": "https://cdn4.vectorstock.com/i/1000x1000/38/23/female-face-avatar-round-flat-icon-with-women-vector-11333823.jpg"
+    "avatar": "https://i.imgur.com/5tYIcRh.png"
```
Comment (makeros) [inline src/public/authors.json:12]: I very much like this change 👍  :)

File: frontend/src/common/constants/sensorsInfo.js (hunk 1 above)
Comment (karlosos) [reply frontend/src/common/constants/sensorsInfo.js:6]: Good point!

**Pull request merged**